db/migrate/20220613110711_migrate_custom_filters.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class MigrateCustomFilters < ActiveRecord::Migration[6.1]
```

Comment:
Would this not be a post-deployment migration?

Reply:
I am not sure why it would be. It won't break anything if the running code is not up-to-date, but it will cause issues if admins forget to run it.